src/components/hooks/use-residence-list.js

```diff
@@ -22,6 +22,7 @@ export const useResidenceList = () => {
                         country.name = el.text
                         country.display_name = el.text
                         country.value = el.value
+                        country.country_code = el.phone_idd
```

Comment:
`el.phone_idd`.  idd, is this a typo here?

Reply:
phone_idd is coming from response

Reply:
Let's refactor this in a new PR, instead of `el.something` let's spread it 

```
response.residence_list.map(({text, value, phone.idd})
```